centaur/src/main/scala/centaur/test/formulas/TestFormulas.scala

```diff
@@ -22,6 +23,35 @@ import spray.json.JsString
   * for comprehension. These assembled formulas can then be run by a client
   */
 object TestFormulas {
+
+  private val jobManagerMetadataRequestArgs = Map(
+    "includeKey" -> List(
+      "attempt",
+      "backendLogs:log",
+      "callCaching:hit",
+      "callRoot",
+      "end",
+      "executionStatus",
+      "failures",
+      "inputs",
+      "jobId",
+      "outputs",
+      "shardIndex",
+      "start",
+      "calls",
+      "description",
+      "executionEvents",
+      "labels",
+      "parentWorkflowId",
+      "returnCode",
+      "status",
+      "submission",
+      "subWorkflowId",
+      "workflowName"
+    ),
+    "excludeKey" -> List("callCaching:hitFailures")
```

Comment:
I wouldn't expect any of the keys with `:` in them to work in Carbonited metadata the way they do with "classic", fixing that is in the vein of BA-6087.